gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -424,7 +427,7 @@ public boolean onBack() {
   @Override
   public boolean onNavigationItemSelected(@NonNull MenuItem item) {
     if (item.getGroupId() == R.id.group_join_project) {
-      projectSelectorViewModel.activateProject(item.getOrder());
+      projectSelectorViewModel.activateLocalProject(item.getOrder());
```

Comment:
We call these "local" here, and "saved" above.  Is there a more consistent name we can use? From the user's perspective, they will "join/leave" a project, which will store them locally for offline use, and sync them when online. Maybe "offline projects" would be clear?